README.md

```diff
@@ -71,6 +71,7 @@ Spacefish is a minimalistic, powerful and extremely customizable <a href="https:
   * `⇕` — diverged chages.
 * Indicator for jobs in the background (`✦`).
 * Current Node.js version, through nvm/nodenv/n (`⬢`).
+* Current Docker version (`🐳`).
```

Comment:
I suggest adding that extra detail. Seems like a useful feature people for people to know about. 😄 

```suggestion
* Current Docker version and connected machine (`🐳`).
```